_departmentpeople/gradstudents/sm3vg.md

```diff
@@ -8,7 +8,7 @@ office: 110 Kerchof Hall
 phone: 434-924-4947
 email: sm3vg@virginia.edu
 image: __SITE_URL__/img/people/2019-fall/Maitra.jpg
-personal_page: https://people.virginia.edu/~sm3vg/
-
+UVa_profile_page: https://profile.virginia.edu/sm3vg
```

Comment:
the replacement ITS has initiated is profile.virginia.edu -- these are single page websites